src/EnergyPlus/CurveManager.cc

```diff
@@ -120,16 +120,14 @@ namespace CurveManager {
     void BtwxtMessageCallback(const Btwxt::MsgLevel messageType, const std::string message, void *contextPtr)
```

Comment:
Passing `std::string` by value?

Reply:
Yep, it's the `btwxt` signature.  I've already discussed with @nealkruis and they have some improvements coming for btwxt anyway.  This will get updated then.